src/paths.ts

```diff
@@ -53,7 +53,11 @@ export function convertClientPathToDebugger(localPath: string, pathMapping?: { [
     let localSourceRoot: string | undefined
     let serverSourceRoot: string | undefined
     let localFileUri = fileUrl(localPath, { resolve: false })
+    let localFileUriLower = fileUrl(localPath.replace(/[a-zA-Z]:\\/, (match: any) => match.toLowerCase()), {
+        resolve: false,
+    })
     let serverFileUri: string
+    let uriWasLowerCased = false
```

Comment:
Frankly I am confused why this fix requires this variable - shouldn't it be possible to just _normalise_ everything to a canonical representation?

Reply:
E.g. before the path mapping feature I had this PR: https://github.com/felixfbecker/vscode-php-debug/pull/198
Can't the same be done, but for all paths in the map?